libs/gretty/src/main/groovy/org/akhikhl/gretty/AppStartTask.groovy

```diff
@@ -25,6 +27,144 @@ class AppStartTask extends StartBaseTask {
   @Delegate
   private WebAppConfig webAppConfig = new WebAppConfig()
 
+  @Input
+  @Optional
+  getContextPath() {
+    webAppConfig.contextPath
+  }
+
+  @Input
+  @Optional
+  getInitParameters() {
+    webAppConfig.initParameters
+  }
+
+  @Input
+  @Optional
+  getRealm() {
+    webAppConfig.realm
+  }
+
+  @Input
+  @Optional
+  getRealmConfigFile() {
+    webAppConfig.realmConfigFile
+  }
+
+  @Input
```

Comment:
Files shouldn't be `@Input`, but `@InputFile` with a `PathSensitivity`.

Reply:
Thanks, will fix that.